confignode/src/main/java/org/apache/iotdb/confignode/manager/ClusterSchemaManager.java

```diff
@@ -354,25 +354,15 @@ public TGetAllTemplatesResp getAllTemplates() {
     resp.setStatus(templateResp.getStatus());
     if (resp.getStatus().getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
       if (templateResp.getTemplateList() != null) {
-        List<ByteBuffer> list = new ArrayList<ByteBuffer>();
-        templateResp
-            .getTemplateList()
-            .forEach(
-                template -> {
-                  list.add(template.serialize());
-                });
+        List<ByteBuffer> list = new ArrayList<>();
+        templateResp.getTemplateList().forEach(template -> list.add(template.serialize()));
         resp.setTemplateList(list);
       }
     }
     return resp;
   }
 
-  /**
-   * show nodes in schema template
-   *
-   * @param req
-   * @return
-   */
+  /** show nodes in schema template */
```

Comment:
why ?   Generally, comments must contain parameter and return value descriptions.

Reply:
But this annotation doesn't interprete the meaning of parameter and return value. I've left TODO for the relevant schoolmates and hope they'll add more detailed instructions later.